travvy-server/src/controllers/RecommendController.js

```diff
@@ -40,10 +41,10 @@ module.exports = {
     return attractions
     }
     else{
-      const activity_ids = await Users_type.findAll({
+      const activity_ids = await Users.findAll({
```

Comment:
What is the difference between these two? Will removing the Users_type impact anything else in the code?

Reply:
The users table is populated upon sign up and onboarding, and contains all the user information. Users_type was going to store the preferred activity type with the activity id, and be used for the recommendation algorithm. After many hours of work, it was found best to just add the activity id to the users table and not use the users_type table. Changes were made in the rest of the code to ensure the changes made do not affect any other functionalities!